docs/semgrep-appsec-platform/jira.md

```diff
@@ -23,72 +23,137 @@ Object.entries(frontMatter).filter(
 
-:::info Prerequisites
-* You must have a **Jira Cloud** plan. Jira Data Center (self-managed, or on-prem) is not supported. 
-* Your Jira user must be able to create a Jira API key with admin level privileges.
-:::
+## Enable the Jira integration
 
-To integrate Jira, follow these steps:
+To enable the Jira integration, follow these steps:
 
-1. In [Semgrep AppSec Platform](https://semgrep.dev/login), go to **Settings** > **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)**.
-2. On the **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)** page, click **Add Integration** (or **Setup First Integration** if this is your first integration), and then select **Ticketing > Jira**.
-3. Follow the on-screen instructions to add your Jira API key.
+1. Sign in to [Semgrep AppSec Platform](https://semgrep.dev/login). 
+2. Navigate to **Settings** > **[Integrations](https://semgrep.dev/orgs/-/settings/integrations)**.
+3. If this is your first integration, click **Setup First Integration**. Otherwise, click **Add integration**. In the drop-down menu that appears, select **Jira**.
+4. Follow the on-screen instructions to grant Semgrep the necessary permissions and set up the integration.
+5. Select the Jira instance you want to connect to when prompted. Choose one instance from the **Use app on** drop-down menu if you have multiple Jira instances.
+   * **For deployments that have used a previous version of the Jira integration**: Ensure you're connecting to the same Jira instance you previously connected to. Please get in touch with Semgrep if you want to connect to a different Jira instance.
 
-## Configure your default project and ticket type
+## Configure the integration
 
-Use the drop-down lists to select the default project and ticket type. These settings can be changed later from the integrations page.
+Once you have enabled the Jira integration, you must complete the following steps on Semgrep AppSec Platform's **Integrations** page:
 
-![Jira configuration modal](/img/jira-configure-defaults.png) 
+1. Confirm the subdomain for your Jira instance
+2. Select the **Default project** where Jira will add the tickets
+3. Select the **Issue type** of the tickets to be created
 
-**Figure** A sample Jira integration
+Click **Save changes** to proceed.
 
-## Create tickets
+![Jira configuration screen](/img/jira-subdomain.png)
+**_Figure._** The Jira configuration screen.
 
-### Code
+### Create mappings
+
+Optionally, you can customize the Jira field data mappings and indicate which Semgrep fields they should be populated with. Currently, Semgrep supports the mapping of the following issue types:
+
+* Short texts
+* Paragraphs
+* Drop-down menus
+* Checkboxes
+
+![Jira configuration screen for field data mappings](/img/jira-configure-defaults.png)
+**_Figure._** The Jira configuration screen for field data mappings.
+
+Semgrep also supports the use of custom issue types and custom fields.
```

Comment:
Nit: Maybe something like: "The integration supports the use of custom Jira issue types and custom fields."